Src/CSharpier.VSCode/src/CSharpierProcessProvider.ts

```diff
@@ -67,15 +67,35 @@ export class CSharpierProcessProvider implements Disposable {
 
         if (!version || !this.csharpierProcessesByVersion[version]) {
             // this shouldn't really happen, but just in case
+            this.logger.debug("returning NullCSharpierProcess");
             return new NullCSharpierProcess();
```

Comment:
If the only reason `NullCSharpierProcess` exists is to make the type checker happy, you can just throw an exception instead.

Reply:
`NullCSharpierProcess` is supposed to be for preventing errors from bubbling up to the user over and over. Although I just tested it out, and it seems like vscode eats any exceptions and doesn't bubble them up to the user at all.